src/main/java/com/odysseusinc/arachne/portal/repository/BaseStudyRepository.java

```diff
@@ -77,4 +77,7 @@
 
     @Query(nativeQuery = true, value = "SELECT * FROM studies_data WHERE id IN :studyIds")
     List<T> findByIdsInAnyTenant(@Param("studyIds") Collection<Long> ids);
+
+    @Query("SELECT s, u FROM Study s JOIN UserStudy u ON s.id = u.study.id WHERE s.kind = com.odysseusinc.arachne.portal.model.StudyKind.WORKSPACE AND u.user.id = :userId")
```

Comment:
As to me I would write smth like: 'SELECT s, u FROM UserStudy u JOIN u.study s WHERE ...'
this just a comment, your version will work